src/components/investors/HomeComponent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import { CredixButton } from '../shared/buttons/CredixButton';
+import SvgIcon, { SVG_ICONS } from '../shared/svg-icon/SvgIcon';
+import { IWidthProps } from '../../pages';
+import { StaticImage } from 'gatsby-plugin-image';
+
+// todo: fix url for becoming investor button
+export const HomeComponent = ({ isMobile }: IWidthProps) => {
```

Comment:
There is already a HomeComponent, rename to InvestorsHomeComponent or so
```suggestion
export const InvestorsHomeComponent = ({ isMobile }: IWidthProps) => {
```